engines/gob/detection/tables_adibou2.h

```diff
@@ -190,7 +190,7 @@
 {
 	{
 		"adibou2",
-		"ADIBÙ 2",
+		"ADIBÙ 2.13 Environment",
```

Comment:
Actually, the Environment disc is called "Ambiente" in the Italian version.
See [here](https://ia601507.us.archive.org/view_archive.php?archive=/20/items/pc.-game-edu-a-scuola-con-adibu-imparo-a-leggere-e-a-contare-prima-elementare/%5BPC.GAME-EDU%5D%20A%20scuola%20con%20adib%C3%B9%20-%20Imparo%20a%20leggere%20e%20a%20contare%20-%20Prima%20elementare.zip&file=%5BPC.GAME-EDU%5D%20A%20scuola%20con%20adib%C3%B9%20-%20Imparo%20a%20leggere%20e%20a%20contare%20-%20Prima%20elementare%2Fcover%2Fcover%20disco%20ambiente.jpg)

Reply:
Good point. @CrystalBen can you also indicate me the name of your German "Environment" disk ?